scripts/las2grain.py

```diff
@@ -24,8 +24,8 @@ def calc_transmittance(altitude_km,
                        lrt_dir='/Users/brent/Code/ice-road-copters/test', 
```

Comment:
@jomey agreed. But for the time being i didn't want to add more user inputs as I'm the only person running this. Ideally this would be another **optional** user input.